src/components/MessageList/MessageNotification.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, MouseEventHandler } from 'react';
+
+export interface MessageNotificationProps {
```

Comment:
we haven't been using interface at all because it can lead to some unknown merging types that we don't necessarily want
```suggestion
export type MessageNotificationProps = {
```